### DISCUSSION TITLE
Fire a callback when a percentage of jobs was run once

### DISCUSSION THREAD

**Discussion by macharmi** [sidekiq/sidekiq#6464] [Q&A]
I need to process millions of records without flooding the queues. So, I am sending a batch of 100 jobs and in the callback, I send another 100 jobs. The problem is that slow jobs are blocking other jobs from being enqueued. Is it possible to fire a callback when 90% of the jobs were enqueued?

**Comment by mperham** [comment]
Have you considered adjusting your queue setup so that those records use a lower-priority queue?
